src/aggregation.py

```diff
@@ -11,21 +13,37 @@ def get_organization_catalog_list():
         for organization in organizations:
```

Comment:
Burde ikke dette og være en async loop? Om jeg leser dette korrekt så venter den til alt er hentet for 1 organisasjon før den begynner på neste...

Reply:
Jo, det burde det definitivt!